server/sonar-server/src/main/java/org/sonar/server/user/ws/UpdateAction.java

```diff
@@ -100,11 +100,13 @@ public void define(WebService.NewController controller) {
   public void handle(Request request, Response response) throws Exception {
     userSession.checkLoggedIn().checkPermission(GlobalPermissions.SYSTEM_ADMIN);
     UpdateRequest updateRequest = toWsRequest(request);
-    doHandle(toWsRequest(request));
-    writeResponse(response, updateRequest.getLogin());
+    try (DbSession dbSession = dbClient.openSession(false);) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this extraneous semicolon. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2959)